### PR TITLE
Ufs release v1.0

### PR DESCRIPTION
**Pull request by jedwards4b**
Changes on the ufs release branch relative to master, most are in config/ufs and will not have any effect on cesm or e3sm

Test suite: scripts_regression_tests.py
Test baseline: 
Test namelist changes: 
Test status: bit for bit, 

Fixes 

User interface changes?: 

Update gh-pages html (Y/N)?:

Code review:

### CODE REVIEW

File: config/cesm/machines/config_machines.xml [hunk 1]
```diff
@@ -370,7 +370,7 @@ This allows using a different mpirun command to launch unit tests
     <DIN_LOC_ROOT_CLMFORC>/glade/p/cgd/tss/CTSM_datm_forcing_data</DIN_LOC_ROOT_CLMFORC>
     <DOUT_S_ROOT>$CIME_OUTPUT_ROOT/archive/$CASE</DOUT_S_ROOT>
     <BASELINE_ROOT>$ENV{CESMDATAROOT}/cesm_baselines</BASELINE_ROOT>
-    <CCSM_CPRNC>$ENV{CESMDATAROOT}/tools/cime/tools/cprnc/cprnc.cheyenne</CCSM_CPRNC>
+<!--    <CCSM_CPRNC>$ENV{CESMDATAROOT}/tools/cime/tools/cprnc/cprnc.cheyenne</CCSM_CPRNC> -->
```
Comment (jgfouca) [inline config/cesm/machines/config_machines.xml]: Was this change intended? It looks to be for testing the cprnc build changes.

Comment (jedwards4b) [reply config/cesm/machines/config_machines.xml]: no, i'll fix it.

**Comment by jedwards4b** [comment]
@jgfouca I have that other conflict fixed in my local master and can push it when the pr is okayed.

File: config/cesm/machines/config_machines.xml [hunk 2]
```diff
@@ -60,7 +60,6 @@ This allows using a different mpirun command to launch unit tests
     <DIN_LOC_ROOT_CLMFORC>$DIN_LOC_ROOT</DIN_LOC_ROOT_CLMFORC>
     <DOUT_S_ROOT>${CIME_OUTPUT_ROOT}/archive/$CASE</DOUT_S_ROOT>
     <BASELINE_ROOT>${CIME_OUTPUT_ROOT}/cesm_baselines</BASELINE_ROOT>
-    <CCSM_CPRNC>/home/jedwards/cesm_tools/cprnc/cprnc</CCSM_CPRNC>
```
Comment (jgfouca) [inline config/cesm/machines/config_machines.xml]: This looks wrong too?

**Comment by jgfouca** [comment]
OK, then I won't worry about stuff that looks wrong for cesm.

File: config/cesm/machines/config_machines.xml (hunk 2 above)
Comment (jedwards4b) [reply config/cesm/machines/config_machines.xml]: I can put it back, but it was in my personal space, so doesnt removing it make the port more general?
